lib/class.LinfoOutput.php

```diff
@@ -269,14 +277,17 @@ public function htmlOut() {
 	if (!empty($settings['cpu_usage']) && isset($info['cpuUsage']) && $info['cpuUsage'] !== false)
 		$core[] = array($lang['cpu_usage'], $info['cpuUsage'].'%');
 
+	// System Load
+	if (!empty($settings['show']['load']))
```

Comment:
Please remove this. You can already optionally enable load progress bars per core ($settings['cpu_usage']), and the uptime load values are not indicative of "real" load on unix.